qiita_db/analysis.py

```diff
@@ -215,6 +215,22 @@ def create(cls, owner, name, description, from_default=False,
         job.submit()
         return instance
 
+    @classmethod
+    def delete_analysis_artifacts(cls, _id):
```

Comment:
Does it make sense to have this be a classmethod? It looks like the first thing it does is instantiate an instance of the class using _id as the parameter. Wouldn't it be more straightforward to create an instance of said class with _id and then call this method?

Reply:
That's how it was originally, see: https://github.com/qiita-spots/qiita/pull/3445/files#diff-e921e32f252a38ad15c8af69792ae15dfa5ecc2e81937b9511d5511117b530dcL295 ... but IMOO it was one too many steps. What do you think?

Reply:
I'm okay with it if you're okay with it. I agree it looks cleaner when using it.